cmake/graph-compiler.cmake

```diff
@@ -25,7 +25,25 @@ if (NOT DEFINED GRAPH_COMPILER_LIBS)
         set(GC_ENABLE_LEGACY OFF)
         set(GC_ENABLE_BINDINGS_PYTHON OFF)
         set(OV_BUILD_SHARED_LIBS_TMP ${BUILD_SHARED_LIBS})
-        set(BUILD_SHARED_LIBS ON)
+        set(BUILD_SHARED_LIBS OFF)
```

Comment:
Does this work with the installation? There's the cpu library search and installation lower in the function.